commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPAttachmentFileEntryLocalServiceImpl.java

```diff
@@ -587,6 +638,31 @@ protected void checkCPAttachmentFileEntriesByExpirationDate()
 		}
 	}
 
+	protected long getGroupId(long classNameId, long classPK)
+		throws PortalException {
+
+		long cpDefinitionClassNameId = classNameLocalService.getClassNameId(
+			CPDefinition.class);
+		long assetCategoryClassNameId = classNameLocalService.getClassNameId(
```

Comment:
We should really sort these, both the `*ClassNameId` variables and the blocks below.